docs/src/main/sphinx/release/release-443.md

```diff
@@ -0,0 +1,49 @@
+# Release 443 (21 Mar 2024)
+
+## General
+
+* Fix formatting of casts from double or real to varchar when running with
+  non-US locales. ({issue}`21136`)
+* Prevent query failure when partial aggregation over decimals with precision
+  larger than 18 below joins. ({issue}`21099`)
+
+## Delta Lake connector
+
+* Automatically use `timestamp(3)` as type during table creation when other timestamp precision is specified. ({issue}`19336`)
+* Ensure all files are deleted when and native S3 file system support is
```

Comment:
Broken sentence: "are deleted *when and* native S3"

Reply:
Updated all four